.storybook/recipes/CalendarCard/CalendarCard.module.css

```diff
@@ -0,0 +1,59 @@
+@import '../../../src/design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #CALENDAR CARD
+\*------------------------------------*/
+
+/**
+ * Card recipe for displaying calendar data
+ */
+.calendar-card {
+  width: 16rem;
+  justify-content: space-between;
+  gap: var(--eds-size-1);
+  background-color: var(--eds-theme-color-background-neutral-default-hover);
+}
+
+.calendar-card--brand {
```

Comment:
Let's start getting into the habit of commenting our CSS. I need to follow these rules myself 😉